WIP-CHANGELOG.md

```diff
@@ -27,3 +27,4 @@
 - Kibana OIDC logout not redirecting correctly.
 - Getting stuck at selecting tenant when logging in to Kibana.
 - The user fluentd configuration uses its dedicated values for tolerations, affinity and nodeselector.
+- Broken OIDC configuration for the ops Grafana instance has been removed.
```

Comment:
One could argue that this entry should be under the `Removed` header, what do you think?

Reply:
I was thinking that initially as well, but placed it under "Fixed" since the "Removed" header didn't exist.
Added it now that you commented it as well :slightly_smiling_face: